### PR TITLE
Adds full cfp details

### PR DESCRIPTION
**Pull request by annalorimer**

### CODE REVIEW

**Comment by evykassirer** [comment]
the form should be linked at the top (and probably elsewhere too)

File: _pages/cfp.md [hunk 1]
```diff
@@ -1,12 +1,145 @@
...
+5. All applicants are informed if their talk was accepted or rejected.
+
+## Guidelines 
+
+We ask for three things other than your basic info: An abstract, a timeline, and a short answer question. Please keep in mind that these are things you must cover in your proposal, but are not limited to.
+
+**Your submission must follow our [Code of Conduct](/coc/).** 
+
+**Do not include any personally identifying information in your abstract, timeline or answer to the short answer question. This includes:**
+- **Your name.**
```
Comment (arshiamufti) [inline _pages/cfp.md]: i would take out the period at the end and the lowercase the first word

File: _pages/cfp.md [hunk 2]
```diff
@@ -1,12 +1,145 @@
...
+- **Where you work/go to school.**
+- **Any relation to the StarCon organizers or members of the CFP Review Committee.**
+- **Your gender, ethnicity, sexual orientation, religion (or lack of) etc.**
+
+### Abstract 
+
+A brief summary of your talk. The reviewers will use this to determine the relevance of the topic and to get a feel for what the talk will cover.
```
Comment (arshiamufti) [inline _pages/cfp.md]: make "A brief summary of your talk." a full sentence

File: _pages/cfp.md [hunk 3]
```diff
@@ -1,12 +1,145 @@
...
+	- _The Calico Method of SoK_
+	- _The Maine Coon Method of SoK_
+	- _Demo_
+- _5 minutes: Conclusions_
+	- _The Future of SoK_
+	- _How you can incorporate SoK into your team_
+
+### Short Answer Question
+
+The point of this is to get to know you better! Think of this as a friendly conversation rather than an elevator pitch.
```
Comment (arshiamufti) [inline _pages/cfp.md]: nit: s/point/purpose?

File: _pages/cfp.md [hunk 4]
```diff
@@ -1,12 +1,145 @@
...
+
+Example: 
+
+_Maintaining good relationships with your co-workers is vital to both personal and organizational development. In my talk Stepping on Keyboards for Fun and Profit I will cover the basics of the technique and how it can improve relationships with your peers, or your human roommates. Stepping on Keyboards (SoK) is a relatively unknown team-building technique among humans, however it is very well known among the feline community. SoK prioritizes getting in the way of a human while they are sitting at their computer, preferably typing, and demanding that they pay attention to you, thus improving communication skills among any human-cat team. The audience will be introduced to the topic, learn several of the common techniques and see a live demo. I believe that this technique is an untapped resource for most teams and that it has the potential to have a large impact on team-building and team development for small to large scale organizations._
+
+### Timeline 
+
+The point of this is to get a feel for what your talk will be like. Unlike the abstract, this will not be posted on the website if your talk is selected.
```
Comment (arshiamufti) [inline _pages/cfp.md]: nit: s/point/purpose and/or s/feel/idea ?

File: _pages/cfp.md [hunk 5]
```diff
@@ -1,12 +1,145 @@
...
+
+### Timeline 
+
+The point of this is to get a feel for what your talk will be like. Unlike the abstract, this will not be posted on the website if your talk is selected. 
+
+Your timeline should: 
+- Include time estimates to prove that your talk is 15 minutes.
```
Comment (arshiamufti) [inline _pages/cfp.md]: "estimates" and "prove" shouldn't be in the same sentence. `Include time estimates to demonstrate that your talk is 15 minutes long`?

File: _pages/cfp.md (hunk 3 above)
Comment (arshiamufti) [inline _pages/cfp.md]: What exactly is the short answer question? That's not clear to me?

File: _pages/cfp.md [hunk 6]
```diff
@@ -1,12 +1,145 @@
...
+subtitle: StarCon 2018 Call for Proposals
 permalink: /cfp/
 ---
 
-Our call for proposals (CFP) for lightning talks will open on August 14 2017 and close on September 15 2017!
 
-All information about our CFP will be posted here.
+We’re excited to announce that StarCon is having it’s first Call for Proposals (CFP)! We’re looking for folks who have a passion for technology, learning and sharing to submit talks about their favourite piece(s) of technology.
+
+We’ve named this conference StarCon (*Con) for a reason. We want to represent many different people and topics in technology, just like how the regex asterisk symbol represents any character or symbol.
```
Comment (arshiamufti) [inline _pages/cfp.md]: you need to escape the `*` here, like `\*` -- that's why the source diff is italicized.

File: _pages/cfp.md [hunk 7]
```diff
@@ -1,12 +1,145 @@
...
+	
+	a) The CFP Review Committee members review their assigned proposals individually, and give the proposal a score. 
+
+	b) The CFP Review Committee gathers to discuss the proposals and to make a list of potential accepted talks.
+4. Speaker names and contact info are de-anonymized and there is a period for the CFP Review Committee to confirm their choices of proposals to accept.
+5. All applicants are informed if their talk was accepted or rejected.
+
+## Guidelines 
+
+We ask for three things other than your basic info: An abstract, a timeline, and a short answer question. Please keep in mind that these are things you must cover in your proposal, but are not limited to.
```
Comment (arshiamufti) [inline _pages/cfp.md]: asking for a "short answer question"? 🤔

File: _pages/cfp.md (hunk 4 above)
Comment (arshiamufti) [inline _pages/cfp.md]: I think applicants would find it confusing that it says that reviewers will use the abstract to get a "feel for what the talk will cover" and then later says that we'll using the timeline to "get a feel for what your talk will be like". Say something like `The purpose of this is to get an idea of the composition of your talk`?

**Comment by annalorimer** [comment]
@arshiamufti Can we move any nits/wording comments be addressed in the google doc?

**Comment by arshiamufti** [comment]
@annalorimer yeah 👍 my bad, I should have provided this feedback before deadline. I didn't realize the final draft was due yesterday :(

**Comment by annalorimer** [comment]
Re: Lines being 80 chars

I didn't shorten them because we haven't done that anywhere else except for the CoC (and even there it isn't consistent).

**Comment by arshiamufti** [comment]
Hm that's fair. it was noticeable here because the lines are longer in the PR. I'd still like to standardize the markdown in this repo before it grows too big, but that can be done in the scope of another PR.

**Comment by annalorimer** [comment]
Agreed that we should standardize and that it's in the scope of another PR

**Comment by arshiamufti** [comment]
@annalorimer my comments are on the g doc now.

**Pull request merged**